pages/signup.js

```diff
@@ -131,25 +134,19 @@ const Signup = () => {
                 <div className="input-wrapper">
                   <label htmlFor="name">Label Name</label>
```

Comment:
Should be `Label name` and `Artist name` based on sentence casing I see on other input labels.